src/server/game/Loot/LootMgr.cpp

```diff
@@ -445,6 +445,34 @@ void Loot::AddItem(LootStoreItem const& item)
         lootItems.push_back(generatedLoot);
         count -= proto->GetMaxStackSize();
 
+        // In some cases, a dropped item should be visible/lootable only for some players in group
+        bool canSeeItemInLootWindow = false;
+        if (Player* player = ObjectAccessor::FindPlayer(lootOwnerGUID))
+        {
+            if (Group* group = player->GetGroup())
+            {
+                for (GroupReference* itr = group->GetFirstMember(); itr != nullptr; itr = itr->next())
```

Comment:
you can use just `auto` instead of specifying the type every time:

```cpp
auto itr = group->GetFirstMember()
```

Reply:
Thanks for the suggestion, its been updated.